render-test/manifest_parser.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <string>
+#include <vector>
+#include "filesystem.hpp"
+
+class ManifestParser {
+public:
+    static ManifestParser& getInstance() {
+        static ManifestParser instance;
+        return instance;
+    }
+    const std::string getVendorPath() const;
+    const std::string getIntegrationPath() const;
+    const mbgl::filesystem::path getBaseTestPath() const;
```

Comment:
no need to return a const type by value, could be `const mbgl::filesystem::path&`